File: Java/retangulo.java
import java.util.Locale;
import java.util.Scanner;
import static java.lang.Math.pow;
import static java.lang.Math.sqrt;

public class retangulo {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);

        double base, altura;

        System.out.print("Insira a base do retangulo: ");
        base = input.nextDouble();

        System.out.print("Insira a altura do retangulo: ");
        altura = input.nextDouble();

        double area, perimetro, diagonal;
        area = base * altura;
        perimetro = 2 * (base + altura);
        diagonal = sqrt((pow(altura,2) + pow(base,2)));

        System.out.println("Area: " + String.format("%.4f", area));
        System.out.println("Perimetro: " + String.format("%.4f", perimetro));
        System.out.println("Diagonal: " + String.format("%.4f", diagonal));

    }
}

File: Python/retangulo.py
'''Fazer um programa para ler as medidas da base e altura de um retângulo. 
Em seguida, mostrar o valor da area, perimetro e diagonal deste retangulo, com quatro casas decimais, conforme exemplos.'''

import math

base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo: "))

area = float(base * altura)
perimetro = float((2 * base) + (2 * altura))
diagonal = float(math.sqrt(base**2 + altura**2))

print(f"Area: {area:.4f}")
print(f"Perimetro: {perimetro:.4f}")
print(f"Diagonal: {diagonal:.4f}")
